src/jni/env.cpp

```diff
@@ -62,7 +59,7 @@ namespace jni {
         if (exception_check()) {
             exception_describe();
             exception_clear();
-            throw JniError("An exception has occurred!");
+            JVM_CRASH_COND_MSG(true, "An exception has occurred!")
```

Comment:
I agree. But as we need to change this in more places in general, I suggest doing this in a separate PR.

Reply:
I do not agree when I read code.
I'm not for checking exception every time we make a jvm call (which is the case for now).
I agree in debug we should make this check, tool should not crash. But in release mode I think we should think a bit more. Even if it is only a bool check, it is a bool check for every jvm method call.
Thing is our module is here to enable user to bring game logic. We should take the less cpu time possible, so that user have much for its game logic.

Reply:
For now I made it crash only in debug when not in TOOL, otherwise it prints error.